src/søknad/Søknadsdialog.tsx

```diff
@@ -1,14 +1,20 @@
-import React, { FC } from 'react';
+import React, { FC, useEffect } from 'react';
 import { Route, Switch } from 'react-router';
-import OmDeg from './inngangsvilkår/omdeg/OmDeg';
+import OmDeg from './steg/1-omdeg/OmDeg';
 import SendSøknad from './SendSøknad';
```

Comment:
**Diskusjon:**

- Endret mappenavnet til steg i stedet for inngangvilkår, yey or nay?
- har også numerert stegene i mappa slik at de sorteres i riktig rekkefølge, yay or nay?

Reply:
Virker logisk å sortere basert på steg ja!